### DISCUSSION TITLE
Can I keep some formula unlinked, while receiving updates

### DISCUSSION THREAD

**Discussion by equal-l2** [Homebrew/discussions#3289] [Everyday usage]
I have both `uutils-coreutils` and `coreutils` installed, but I don't want to link `coreutils`.
Whenever `coreutils` receives an update, Homebrew tries to link `coreutils` and complains conflict with `uutils-coreutils`.

Can I keep `coreutils` unlinked while receiving its updates?

**Comment by carlocab** [comment]
Not that I know of, but this might be a nice feature to have. PRs to implement it welcome.
